content/docs/user-guide/troubleshooting.md

```diff
@@ -92,16 +92,17 @@ using:
 $ dvc checkout --relink
 ```
 
-## HTTP Git authentication is not supported {#git-auth}
+## DVC can only authenticate with Git remotes using SSH URLs {#git-auth}
 
 [Experiment sharing](/doc/user-guide/experiment-management/sharing-experiments)
-commands accept a `git_remote` argument. In order to access the Git remote, you
-may need to authenticate for _write_ (`dvc exp push`) or _read_ (`dvc exp list`,
+commands accept a `git_remote` argument. You may need to authenticate to use the
+Git remote, for _write_ (`dvc exp push`) or _read_ (`dvc exp list`,
 `dvc exp pull`) permissions.
 
 DVC does not currently support authentication with [Git credentials]. This means
 that unless the Git server allows unauthenticated HTTP write/read, you should
-use an [SSH Git URL] when listing, pulling or pushing experiments.
+use an [SSH Git URL] for Git remotes used for listing, pulling or pushing
+experiments.
```

Comment:
Hello @daavoo can you confirm this change us OK? Thanks

Reply:
I think that it goes back to the original discussion 😅 . The title feels a little misleading for me, because this issue only applies to `experiments` and the new phrase seems to imply that it applies to DVC in general (maybe `DVC experiments` instead of `DVC`?).

The body changes lgtm

Reply:
Agree but the title is already a bit long and users mainly get here from an error printed when using `dvc exp` so shouldn't be a problem really 🙂